ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -453,6 +453,33 @@ export class ParticipantListComponent implements OnInit {
                         }
                       });
                     }
+                  } else if (question.questionType === 'COMPOSITE') {
+                    options = new Array<NameValue>();
+                    type = Filter.OPTION_TYPE;
+                    if (question.childQuestions != null) {
+                      question.childQuestions.forEach((childQuestion: QuestionDefinition) => {
+                        if (childQuestion.options != null) {
+                          childQuestion.options.forEach((option: Option) => {
+                            options.push(new NameValue(option.optionStableId, option.optionText));
+                            if (option?.nestedOptions != null) {
```

Comment:
how about changing these checks to `if (options?.nestedOptions)` to avoid future undefined error